### ISSUE TITLE
Search results in Atom 1.17.2

### ISSUE THREAD

**Issue by quicoto** [jesseweed/seti-ui#427]
As from Atom 1.17.2 the Search Results are somewhat broken.

Using Seti-ui 1.8.0

Thank you!

![Search Results](https://cloudup.com/cAZR_k8jrdL+)

**Comment by persac** [comment]
You may add this CSS rule to the Atom styles.less stylesheet (Atom > Stylesheet…) to reveal the results while waiting for a true fix:
```
.preview-pane .results-view .path.list-nested-item .matches.list-tree .search-result.list-item {
  padding: 0 10px;
}
```

**Comment by jesseweed** [comment]
Thanks guys! Fix for this is coming soon!

**Comment by jesseweed** [comment]
This should be fixed in the 1.9 release. Please let me know if you still experience the issue after updating. Thanks!
